samples/net/mqtt_publisher/boards/cc3220sf_launchxl.conf

```diff
@@ -0,0 +1,26 @@
+# Networking Config:
+CONFIG_NETWORKING=y
```

Comment:
These are alreade set in the same manner in general `prj.conf`, I don't think you need to repeat it here. IMO board specific configs should be kept minimal (if needed at all).

Reply:
I followed the lead from existing examples, but you are right, I can remove this line.